### ISSUE TITLE
train issue

### ISSUE THREAD

**Issue by Kiri0824** [roger-tseng/av-superb#4]
I'm currently training this set of code on a Chinese dataset. The upstream part I'm using is the fusion_feats of avhubert, and the downstream part is av_asr. I've compiled the dictionary based on Chinese. Now I've encountered a problem. When the model was initially initialized, the predicted tokens had values. However, after several training steps, when I was calculating the metrics and outputted the token results of pred, they were all empty.

**Comment by roger-tseng** [comment]
Could you check the training loss over time to see if the performance is actually improving? 

If the training loss isn't improving, it's possible that your model isn't actually fitting the training data very well, since the default configuration for ASR here only trains a small amount of parameters.

To improve the performance, you can try increasing the number of trained parameters by using a larger prediction head, or finetuning the AV-HuBERT encoder with the `--upstream_trainable` option in `run_downstream.py`

**Comment by Kiri0824** [comment]
> Could you check the training loss over time to see if the performance is actually improving?
> 
> If the training loss isn't improving, it's possible that your model isn't actually fitting the training data very well, since the default configuration for ASR here only trains a small amount of parameters.
> 
> To improve the performance, you can try increasing the number of trained parameters by using a larger prediction head, or finetuning the AV-HuBERT encoder with the `--upstream_trainable` option in `run_downstream.py`

here's my result:
![image](https://github.com/user-attachments/assets/ecf483c3-ec10-42e0-b887-e320e9f0eaa3)
loss seems normal. but i print pred token when calculating the metrics. all empty. i think its my configuration problem ?

**Comment by roger-tseng** [comment]
Sorry for the late reply! The training loss in the diagram is way too high, a converging model's training loss should decrease to around 0.001. 

As I mentioned above, Increasing the number of layers or finetuning the encoder parameters may help with this.
